default/python/AI/AIDependencies.py

```diff
@@ -318,11 +305,6 @@ def not_affect_by_special(special: str, species: str) -> bool:
 }
 # </editor-fold>
 
-# <editor-fold desc="Detection related specials">
-PANOPTICON_SPECIAL = "PANOPTICON_SPECIAL"
-PANOPTICON_DETECTION_BONUS = 10
```

Comment:
Looks like another special not evaluated by colonization rating. Please keep it. The bonus should probably become a named value.

Reply:
Keeping this in code want help.  Maybe we should put it to the issue?  I guess there might be other species that missed.